pyiso/base.py

```diff
@@ -18,7 +18,7 @@
     from urllib.request import urlopen  # Changed from urllib2 for python3.x
 
 # named tuple for time period interval labels
-IntervalChoices = namedtuple('IntervalChoices', ['hourly', 'fivemin', 'tenmin', 'fifteenmin', 'na', 'dam'])
+IntervalChoices = namedtuple('IntervalChoices', ['hourly', 'fivemin', 'tenmin', 'fifteenmin', 'na', 'dam', 'allbusrt'])
```

Comment:
allbusrt is a type of node, rather than a type of market.  This should be handled with a special case of node_id, as it is the same market as RT5M